cfg/DynamicRobotState.cfg

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This is not needed (and possibly even wrong in ROS1). catkin automatically transforms shebangs to either python or python3 based on the python version you are building against (check the generated files in devel or install directories).